docs/guides/known-issues.md

```diff
@@ -21,3 +21,17 @@ Workaround:
 
 - Use a SystemBinary with [`SYSTEM_BINARY`](../api/config-options.md#SYSTEM_BINARY)
 - Do not use AlpineLinux
+
+## libcurl on slim Docker images
+
+Docker images of the "slim" variety often don't have `libcurl` installed,
+such as [the official Node.js image](https://hub.docker.com/_/node) or [sitespeed.io node](https://hub.docker.com/r/sitespeedio/node).
+If you don't install it manually, then `mongodb-memory-server` will not start up in your container, with the following exception message:
```

Comment:
```suggestion
Docker images of the "slim" variety often don't have `libcurl` installed, such as [the official Node.js image](https://hub.docker.com/_/node) or [sitespeed.io node](https://hub.docker.com/r/sitespeedio/node).  
If you don't install it manually, then `mongodb-memory-server` will not start up in your container, with the following exception message:
```

The Pages are dynamically rendered, so if it is one sentence, try to keep it in one line until a seperator (like `?`, `!`, `.`, `:`, etc)